src/libraries/System.Net.Sockets/src/System/Net/Sockets/TCPClient.cs

```diff
@@ -283,6 +286,9 @@ private async Task CompleteConnectAsync(Task task)
         public ValueTask ConnectAsync(IPAddress[] addresses, int port, CancellationToken cancellationToken) =>
             CompleteConnectAsync(Client.ConnectAsync(addresses, port, cancellationToken));
 
+        public ValueTask ConnectAsync(EndPoint remoteEP, CancellationToken cancellationToken) =>
+            CompleteConnectAsync(Client.ConnectAsync(remoteEP, cancellationToken));
```

Comment:
I know it's inconsistent with the rest of the file, but we're moving to a model of using XML comments for documentation.  Can you add XML comments for the two new methods?  You can copy liberally from the existing docs (https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.tcpclient.connectasync?view=net-5.0).

Reply:
I added a comment. Hopefully, it's OK.